hudi-client/hudi-client-common/src/main/java/org/apache/hudi/index/bloom/HoodieBloomIndex.java

```diff
@@ -234,8 +234,8 @@ protected List<Pair<String, BloomIndexFileInfo>> loadColumnRangesFromMetaIndex(
           new BloomIndexFileInfo(
               FSUtils.getFileId(entry.getKey().getRight()),
               // NOTE: Here we assume that the type of the primary key field is string
-              (String) unwrapStatisticValueWrapper(entry.getValue().getMinValue()),
-              (String) unwrapStatisticValueWrapper(entry.getValue().getMaxValue())
+              unwrapStatisticValueWrapper(entry.getValue().getMinValue()).toString(),
+              unwrapStatisticValueWrapper(entry.getValue().getMaxValue()).toString()
```

Comment:
Could potentially result in `null` dereference. 
Also, we have made a note of the assumption that primary key is supposed to be string.

Reply:
Thank you very much, I think if the primary key is null, an exception should be thrown inside this method. After I modified it like this, all other reads and writes were normal. Even if the primary key is bigint

Reply:
@codope  Can you tell me what I need to change? thank you.

Reply:
> Could potentially result in null dereference

The primary key could never be null. This is constraint from SQL, +1 to the fix.